devimint/src/tests.rs

```diff
@@ -288,7 +288,7 @@ pub async fn latency_tests(dev_fed: DevFed, r#type: LatencyTest) -> Result<()> {
             let fm_pay_stats = stats_for(fm_internal_pay);
 
             println!("### LATENCY FM PAY: {fm_pay_stats}");
-            assert!(fm_pay_stats.median < Duration::from_secs(12));
+            assert!(fm_pay_stats.median < Duration::from_secs(15));
```

Comment:
Maybe latency tests should not run in parallel on the same machine? This hack may hide latency introduced from round trips that don't require much compute?

Reply:
Good callout 👍 

Made an issue to make sure we keep track https://github.com/fedimint/fedimint/issues/4579.